vowpalwabbit/multilabel.cc

```diff
@@ -135,11 +137,10 @@ void print_update(vw& all, bool is_test, example& ec)
     if (is_test)
       label_string << " unknown";
     else
-      for (size_t i = 0; i < ec.l.multilabels.label_v.size(); i++) label_string << " " << ec.l.multilabels.label_v[i];
+      for (unsigned int i : ec.l.multilabels.label_v) label_string << " " << i;
```

Comment:
Use the same type as is contained
```suggestion
      for (uint32_t i : ec.l.multilabels.label_v) { label_string << " " << i; }
```